event/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+# Create your models here.
+class Event(models.Model):
+    event_id = models.IntegerField()
+    name = models.CharField(max_length=200)
+    activities = models.CharField(max_length=9000)
+    budget = models.IntegerField(default=1)
+
+
+    def save(self):
```

Comment:
usually when you create a new event, you don't want supply the ID into creation method, so I think this method doesn't need to be here

Reply:
Right so supplying the id here was the work around for the auto int not working